sdk/servicebus/service-bus/package.json

```diff
@@ -87,24 +100,26 @@
     "build:browser": "tsc -p . && cross-env ONLY_BROWSER=true rollup -c 2>&1",
     "build:node": "tsc -p . && cross-env ONLY_NODE=true rollup -c 2>&1",
     "build:samples": "node .scripts/prepSamples.js && cd samples && tsc -p .",
-    "build:test": "tsc -p . && cross-env ONLY_NODE=true rollup -c rollup.test.config.js 2>&1",
-    "build": "tsc -p . && rollup -c 2>&1",
+    "build:test:browser": "tsc -p . && cross-env BROWSER_TEST=true rollup -c 2>&1",
+    "build:test:node": "tsc -p . && cross-env ONLY_NODE=true rollup -c rollup.test.config.js 2>&1",
+    "build:test": "npm run build:test:node && npm run build:test:browser",
+    "build": "npm run build:node && npm run build:browser && npm run build:test:node && npm run build:test:browser",
     "check-format": "prettier --list-different --config .prettierrc.json \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
     "clean": "rimraf dist dist-esm test-dist typings *.tgz *.log",
     "coverage": "nyc --reporter=lcov --exclude-after-remap=false mocha -t 120000 test-dist/index.js --reporter mocha-multi --reporter-options spec=-,mocha-junit-reporter=-",
     "extract-api": "tsc -p . && api-extractor run --local",
     "format": "prettier --write --config .prettierrc.json \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
-    "integration-test:browser": "echo skipped",
+    "integration-test:browser": "karma start --single-run",
     "integration-test:node": "mocha -t 120000 test-dist/index.js --reporter mocha-multi --reporter-options spec=-,mocha-junit-reporter=-",
     "integration-test": "npm run integration-test:node && npm run integration-test:browser",
     "lint:fix": "tslint -p . -c tslint.json --fix",
     "lint": "tslint -p . -c tslint.json",
     "pack": "npm pack 2>&1",
     "prebuild": "npm run clean",
     "preinstall": "node ../../../common/scripts/rush-welcome.js",
-    "test:browser": "npm run build:test && npm run unit-test:browser && npm run integration-test:browser",
-    "test:node": "npm run build:test && npm run unit-test:node && npm run integration-test:node",
-    "test": "npm run build:test && npm run unit-test && npm run integration-test",
+    "test:browser": "npm run integration-test:browser",
+    "test:node": "npm run integration-test:node",
+    "test": "npm run test:node && npm run test:browser",
```

Comment:
Probably simpler as:
`"test": "npm run build:test && npm run integration-test",`

That way `build:test` only runs once

Reply:
But I thought we wanted to remove build:test out of the test commands as build system already executes it?

Reply:
The build system only knows about the "integration-test" and "unit-test" targets. And yes, it will run "build:test" before it executes either of those targets.

However the "test" targets are there for the developer's inner loop, the build system doesn't know or care about them. Hence why I suggest maybe adding that back in to just the "test" targets.